{{cookiecutter.repo_name}}/circle.yml

```diff
@@ -1,24 +1,83 @@
 # CircleCI Configuration file
 
+
 machine:
-  python:
-    version: 3.6.2
+  pre:
+    # we love jq
+    - sudo apt-get update && sudo apt-get install -yq jq
+    # install a reasonable version of Docker
+    - sudo curl -sSL https://s3.amazonaws.com/circle-downloads/install-circleci-docker.sh | bash -s -- 1.10.0
+  services:
+    - docker
 
-general:
-  artifacts:
-     - "dist"
-     - "cover"
 
 dependencies:
   override:
-    - pip install tox tox-pyenv wheel
-    - pyenv local 3.6.s
+    # make sure docker is running
```

Comment:
We actually have two repos for Python services. The main one (this one) is public and tries to avoid Globality-isms, especially related to builds. The other one (cookiecutter-python-build) is private and contains only the build related files.

So we don't really want this here. We could just remove `circle.yml` entire.

Reply:
Got it. I wasn't sure if this was geared for internal use or not (meaning taking docker and the like into consideration here).